cli/compose/schema/data/doc.go

```diff
@@ -0,0 +1,4 @@
+//
+// +domain=docker.com
+
+package data
```

Comment:
As it is `doc.go` maybe we can add a comment of what's in there and how to use it with `docker/cli/cli/compotse` :stuck_out_tongue: